pkg/config/config.go

```diff
@@ -142,6 +142,16 @@ type PluginConfig struct {
 	// Log line longer than the limit will be split into multiple lines. Non-positive
 	// value means no limit.
 	MaxContainerLogLineSize int `toml:"max_container_log_line_size" json:"maxContainerLogSize"`
+	// NoCgroup indicates to disable the cgroup support. (auto/true/false)
+	// This is useful when the daemon does not have permission to access cgroups.
+	NoCgroup string `toml:"no_cgroup" json:"noCgroup"`
```

Comment:
s/NoCgroup/DisableCgroup.

We use `enable_xxx` for the other options, let's follow the same pattern.